src/test/mainnet/AaveV3EthAddCRVPoolPayloadTest.t.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.17;
+
+import 'forge-std/Test.sol';
+import {AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {ProtocolV3TestBase, InterestStrategyValues, ReserveConfig} from 'aave-helpers/ProtocolV3TestBase.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {TestWithExecutor} from 'aave-helpers/GovHelpers.sol';
+import {AaveV3EthAddCRVPoolPayload} from '../../contracts/mainnet/AaveV3EthAddCRVPoolPayload.sol';
+
+contract AaveV3EthAddCRVPoolPayloadTest is ProtocolV3TestBase, TestWithExecutor {
```

Comment:
Should we have some integration tests checking borrowing, depositing, etc. post-listing?

Reply:
@sakulstra had mentioned last week that they don't think that's necessary for new listings as the protocol itself has been heavily tested.

Reply:
SGTM